app/ide-desktop/lib/dashboard/src/services/LocalBackend.ts

```diff
@@ -5,13 +5,82 @@
  * the API. */
 import * as detect from 'enso-common/src/detect'
 
+import * as appUtils from '#/appUtils'
+
 import Backend, * as backend from '#/services/Backend'
 
 import * as dateTime from '#/utilities/dateTime'
 import * as errorModule from '#/utilities/error'
+import * as fileInfo from '#/utilities/fileInfo'
 import * as projectManager from '#/utilities/ProjectManager'
 import ProjectManager from '#/utilities/ProjectManager'
 
+// =============
+// === Types ===
```

Comment:
My understanding of the style guide is that headers should match the name of the item they precede. So we should have a `ProjectMetadata` header, an `Attributes` header, etc.

They're not meant to denote "sections" of the code, but to help find items on a quick scroll.

Reply:
hm. you might be right...
<https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md#sections>
i think it's out of scope for this PR though, since i've been using headers for `Types` and `Constants` pretty liberally.

also one issue with that approach, is that in the dashboard, it's more or less one "concept"/main structure per file. most of the exceptions are in utility modules that export a bunch of functions instead of a class
so that means these "sections" would be kinda redundant